setup.py

```diff
@@ -62,7 +109,7 @@
         "pillow",
         (
             "compressed-tensors==0.9.3"
-            if version_info.build_type == "release"
+            if BUILD_TYPE == "release"
             else "compressed-tensors-nightly"
```

Comment:
Since we'll apply the same changes to compressed-tensors as well, should we update here to install only compressed-tensors?

Reply:
That's a good point. Maybe we should make the change in the other repository before making it here?

Reply:
Yeah I'm working on it, hopefully to have it ready shortly.

Reply:
OK, I've updated this code to not change the package name but to still change the requirement so that it is not using a pinned version of the compressed-tensors package. It seems like that will let you have the control you want for releases, then be more flexible in other situations. Let me know if you had something different in mind.

Reply:
Thanks, the updated code looks good to me:

        "pillow",
        (
            "compressed-tensors==0.9.3"
            if version_info.build_type == "release"
            else "compressed-tensors-nightly"
            if BUILD_TYPE == "release"
            else "compressed-tensors"
        ),

@dsikka can you please review if this looks correct to you?